src/main/java/com/minecolonies/entity/ai/citizen/builder/EntityAIStructureBuilder.java

```diff
@@ -579,29 +586,32 @@ private AIState decorationStep()
         return findNextBlockNonSolid();
     }
 
-    private void spawnEntity(Entity entity)//TODO handle resources
+    //TODO handle resources
+    private void spawnEntity(Entity entity)
     {
-        if(entity != null)
+        if (entity != null)
         {
             BlockPos pos = job.getSchematic().getOffsetPosition();
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Local variable 'pos' could be declared final [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://home.kk-sc.de:9000/coding_rules#rule_key=pmd%3ALocalVariableCouldBeFinal)